zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -85,12 +87,15 @@ object HttpData {
   /**
    * Helper to create HttpData from contents of a file
    */
-  def fromFile(file: java.io.File): HttpData = File(file)
+  def fromFile(file: java.io.File): HttpData = {
+    val raf = new java.io.RandomAccessFile(file, "r")
+    RandomAccessFile(raf)
+  }
 
   private[zhttp] final case class Text(text: String, charset: Charset)                   extends HttpData
   private[zhttp] final case class BinaryChunk(data: Chunk[Byte])                         extends HttpData
   private[zhttp] final case class BinaryByteBuf(data: ByteBuf)                           extends HttpData
   private[zhttp] final case class BinaryStream(stream: ZStream[Any, Throwable, ByteBuf]) extends HttpData
-  private[zhttp] final case class File(file: java.io.File)                               extends HttpData
+  private[zhttp] final case class RandomAccessFile(raf: java.io.RandomAccessFile)        extends HttpData
```

Comment:
Is it possible to use the base type for `RandomAccessFile`?

That way we will be able to support more types.

Reply:
I don't think this will work. We need a `FileChannel` later ,which we can't get from the base type of `RandomAccessFile`